bpf/go_nethttp.h

```diff
@@ -324,10 +386,8 @@ static __always_inline void roundTripStartHelper(struct pt_regs *ctx) {
 
     http_func_invocation_t invocation = {.start_monotime_ns = bpf_ktime_get_ns(), .tp = {0}};
 
-    __attribute__((__unused__)) u8 existing_tp = client_trace_parent(
-        goroutine_addr,
-        &invocation.tp,
-        (void *)(req + go_offset_of(ot, (go_offset){.v = _req_header_ptr_pos})));
+    __attribute__((__unused__)) u8 existing_tp =
```

Comment:
do we still need this attribute xor `existing_tp`?

Reply:
Nope we can clean up.